Mathlib/Order/RelSeries.lean

```diff
@@ -205,6 +212,8 @@ def head (x : RelSeries r) : α := x 0
 Since a relation series is assumed to be non-empty, this is well defined. -/
 def last (x : RelSeries r) : α := x <| Fin.last _
 
+lemma last_def (x : RelSeries r) : x (Fin.last <| x.length) = x.last := rfl
```

Comment:
`foo_def` should be of the form `foo = _`, not `_ = foo`

Reply:
I changed the lemma to `apply_last`